### ISSUE TITLE
Errors on insert with parameter markers

### ISSUE THREAD

**Issue by data-henrik** [nakagami/pydrda#15]
I am trying to run an INSERT statement that has string, integer, and date values. The statement works when passing the values as literals, but fails when trying to use parameter markers (`?`).

Could such a scenario be added to the Db2 unit tests, both for QA and serving as example?

**Comment by nakagami** [comment]
The test around here 
https://github.com/nakagami/pydrda/blob/master/test_db2.py#L130-L137
confirms the operation of select and update. I think insert will probably work as well.

**Comment by data-henrik** [comment]
That is one of the errors:
```
  File "/home/user/.local/lib/python3.11/site-packages/drda/connection.py", line 367, in _query
    rows, _, _ = self._parse_response()
                 ^^^^^^^^^^^^^^^^^^^^^^
  File "/home/user/.local/lib/python3.11/site-packages/drda/connection.py", line 102, in _parse_response
    raise err
drda.OperationalError: -180:b'22007':b''
```

**Comment by data-henrik** [comment]
another error when just using two string parameter markers on INSERT
```
  File "/home/hloeser/.local/lib/python3.11/site-packages/drda/connection.py", line 102, in _parse_response
    raise err
drda.OperationalError: -30020:b'58009':b'0x220E"("0133")"\xff'
```

**Comment by nakagami** [comment]
I would appreciate it if you could provide an example of an insert statement for the table definition to be reproduced.

I see here that you are getting errors with date, datetime, and timestamp.
https://stackoverflow.com/questions/14959872/insert-throws-sqlcode-180-sqlstate-22007-on-ibm-db2

How about something like
```
insert into foo (column) values (timestam(?))
```

**Comment by nakagami** [comment]
Thank you.
I was able to write a test that produces the error
https://github.com/nakagami/pydrda/commit/50d5ff93f48194c51541c51783d2a9df598327aa

Hmmm, I don't know the cause, but it looks like there is something wrong with the encoding when passing string parameters.

Maybe it's something around here
https://github.com/nakagami/pydrda/blob/master/drda/ddm.py#L420
I don't know how to fix it yet.

**Comment by nakagami** [comment]
@data-henrik

**Comment by data-henrik** [comment]
@nakagami Thank you! I plan to test it today, Monday latest and let you know.

**Comment by nakagami** [comment]
Thanks, fixed version has been released as 0.4.8
